src/api/app/views/webui/package/show.html.erb

```diff
@@ -142,7 +142,7 @@
             <%# TODO: only users w/o rights should see this, maintainers should get a different dialog: %>
             <% if @package.develpackage %>
                 <li>
-                  <%= link_to(sprited_text('arrow_switch', 'Request devel project change'), { :controller => 'request', :action => 'change_devel_request_dialog', :project => @project, :package => @package }, :remote => true) %>
+                  <%= link_to(sprited_text('arrow_switch', 'Request devel project change'), { controller: :request, action: :change_devel_request_dialog, project: @project, package: @package }, remote: true) %>
```

Comment:
Why don't you set up a named routing helper?

Reply:
because the names are messed up in the route file... I would do it in a separate commit and change what is not Rails like. I asked to remove them before because of that.